### PR TITLE
WIP: add serial protocol for SBUS1 output

### PR DESCRIPTION
**Pull request by kd0aij**
probably needs some rework and could be implemented for board types other than PX4

### CODE REVIEW

**Comment by kd0aij** [comment]
test with FrSky Sbus to PWM converter on telem2 of a PixRacer.
note that sbus frame rate is 50Hz, but PWM output rate is 167Hz:

sbus_pwm

File: ArduPlane/servos.cpp [hunk 1]
```diff
@@ -732,10 +732,65 @@ void Plane::set_servos(void)
...
+void
+Plane::sbus1_out(AP_HAL::UARTDriver* uart)
+{
+    uint8_t buffer[SBUS_BSIZE] = { 0x0f };  // first byte is always 0x0f
+    uint8_t index = 1;
+    uint8_t offset = 0;
+    uint16_t value;
+
+    /* construct sbus frame representing channels 1-16  */
+    for (unsigned i = 0; i < SBUS_CHANNELS; ++i) {
```
Comment (rmackay9) [inline ArduPlane/servos.cpp]: "unsigned" should probable "unit16_t"

File: ArduPlane/servos.cpp [hunk 2]
```diff
@@ -732,10 +732,65 @@ void Plane::set_servos(void)
     }
 
     // run output mixer and send values to the hal for output
+    static uint64_t last_time = 0;
+    if (last_time == 0) last_time = AP_HAL::micros64();
+    else {
+        uint64_t now = AP_HAL::micros64();
+        uint32_t delta_t = now - last_time;
+        last_time = now;
+        ::printf("dt: %d\n", delta_t);
```
Comment (rmackay9) [inline ArduPlane/servos.cpp]: probably good to remove this debug

**Comment by kd0aij** [comment]
To make this work for all boards, method Plane::sbus1_out() in servos.cpp, needs to move down, perhaps into AP_HAL::RCOutput?

**Comment by kd0aij** [comment]
superseded by #6794

**Pull request closed without merging**